qemu/tests/uefi_pkg.py

```diff
@@ -0,0 +1,70 @@
+import re
+import json
+import logging
+
+from avocado.utils import process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify descriptor meta-files after ovmf package installation on the host:
+    1) Check edk2-ovmf package has been installed already.
+    2) Qurey file list on edk2-ovmf package.
+    3) Make sure the descriptor meta-files will be in file list.
+       2 files in total.
+       For rhel7: 50-ovmf-sb.json, 60-ovmf.json
+       For rhel8: 40-edk2-ovmf-sb.json, 50-edk2-ovmf.json
+    4) Check the JSON files internally.
+       check that the "filename" elements in both files point to valid files.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def check_element(filename, file_list):
+        """
+        check 'filename' element point to a valid file
+
+        :param filename: 'filename' element
+        :param file_list: query files command output
+        """
+        if filename not in file_list:
+            test.fail("The 'filename' element in meta-file point to an "
+                      "invalid file. The invalid file is '%s'" % filename)
+
+    query_package = params["query_package"]
+    error_context.context("Check edk2-ovmf package has been "
+                          "installed already", logging.info)
+    status, output = process.getstatusoutput(query_package,
+                                             ignore_status=True,
+                                             shell=True)
+    if status:
+        test.error("Please install edk2-ovmf package on host.")
+    package_name = params["ovmf_package_name"]
+    ovmf_package = re.findall(package_name, output, re.S)
+    if not ovmf_package:
```

Comment:
use framework params "ovmf_path" instead check package name.

Reply:
ovmf_path is not enough.
# ls /usr/share/edk2/ovmf/
EnrollDefaultKeys.efi  OVMF_CODE.secboot.fd  OVMF_VARS.fd  OVMF_VARS.secboot.fd  Shell.efi  UefiShell.iso

need get correct package name to query and get meta-files.
e.g.
# rpm -qa|grep edk2
edk2-tools-20190829git37eef91017ad-8.el8.x86_64
edk2-ovmf-20190829git37eef91017ad-8.el8.noarch
edk2-tools-debuginfo-20190829git37eef91017ad-8.el8.x86_64
edk2-debugsource-20190829git37eef91017ad-8.el8.x86_64

# rpm -ql edk2-ovmf-20190829git37eef91017ad-8.el8.noarch